src/Compilers/CSharp/Test/Semantic/Semantics/RecordStructTests.cs

```diff
@@ -5190,5 +5156,959 @@ public void ToString_UserDefinedPrintMembers_Static()
                 Diagnostic(ErrorCode.ERR_StaticAPIInRecord, "PrintMembers").WithArguments("C1.PrintMembers(System.Text.StringBuilder)").WithLocation(4, 25)
                 );
         }
+
+        [Fact]
+        public void AmbigCtor_WithFieldInitializer()
```

Comment:
>AmbigCtor_WithFieldInitializer [](start = 20, length = 30)

It is not clear why the test is called this. It doesn't look like the scenario has any ambiguity.

Reply:
This is a ported test from record classes where it is an ambiguity. I kept the matching test names when moving them.

---
In reply to: [605132581](https://github.com/dotnet/roslyn/pull/52256#discussion_r605132581) [](ancestors = 605132581)